openghg/standardise/footprint/_paris.py

```
@@ -0,0 +1,175 @@
+    Returns:
+        dict: Dictionary of data
+    """
+
+    if high_time_resolution:
+        warnings.warn(
+            "This argument is deprecated and will be replaced in future versions with time_resolved.",
+            DeprecationWarning,
+        )
+        time_resolved = high_time_resolution
+
+    xr_open_fn, filepath = check_function_open_nc(filepath)
+
+    with xr_open_fn(filepath) as temp:
+        fp_data = temp
+
+    time_resolved = check_species_time_resolved(species, time_resolved)
+    short_lifetime = check_species_lifetime(species, short_lifetime)
+
+    dv_rename = {"srr": "fp"}
+
+    attribute_rename = {"fp_output_units": "LPDM_native_output_units"}
+
+    dim_rename = {"latitude": "lat", "longitude": "lon"}
+
+    dim_reorder = ("time", "height", "lat", "lon")
+
+    # Ignore type - dv_rename type should be fine as a dict but mypy unhappy.
+    fp_data = fp_data.rename(**dv_rename)  # type: ignore
+    fp_data = fp_data.rename(**dim_rename)
+
+    fp_data = fp_data.transpose(*dim_reorder, ...)
+
+    for attr, new_attr in attribute_rename.items():
+        if attr in fp_data:
+            fp_data.attrs[new_attr] = fp_data.attrs.pop(attr)
+
+    # Need to read the metadata from the footprints and then store it
+    # Do we need to chunk the footprints / will a Datasource store it correctly?
+    metadata: Dict[str, Union[str, float, List[float]]] = {}
+
+    metadata["data_type"] = "footprints"
+    metadata["site"] = site
+    metadata["domain"] = domain
+    metadata["model"] = model
+
+    # Include both inlet and height keywords for backwards compatability
+    metadata["inlet"] = inlet
+    metadata["height"] = inlet
+    metadata["species"] = species
+
+    if met_model is not None:
+        metadata["met_model"] = met_model
+
+    if network is not None:
+        metadata["network"] = network
+
+    # Check if time has 0-dimensions and, if so, expand this so time is 1D
```

If a footprint doesn't have a time coordinate, should we raise a value error?

---

Added an else to current if statement for this to raise a `ParseError`